src/test/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltrationIntegrationTest.java

```diff
@@ -0,0 +1,246 @@
+                Arrays.asList(getToolTestDataDir() + "expected/" + "testVariantFiltration_testGenotypeFilters2.vcf")
+        );
+
+        spec.executeTest("test file", this);
+    }
+
+    @Test
+    public void testDeletions() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString("twoDeletions.vcf", " --filterExpression 'QUAL < 100' --filterName foo "),
+                Arrays.asList(getToolTestDataDir() + "expected/" + "testVariantFiltration_testDeletions.vcf")
+        );
+
+        spec.executeTest("test file", this);
+    }
+
+    @Test
+    public void testUnfilteredBecomesFilteredAndPass() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString("unfilteredForFiltering.vcf", " --filterExpression 'FS > 60.0' --filterName SNP_FS "),
+                Arrays.asList(getToolTestDataDir() + "expected/" + "testVariantFiltration_testUnfilteredBecomesFilteredAndPass.vcf")
+        );
+
+        spec.executeTest("test file", this);
+    }
+
+    @Test
+    public void testFilteringDPfromINFO() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString("filteringDepthInFormat.vcf", " --filterExpression 'DP < 8' --filterName lowDP "),
+                Arrays.asList(getToolTestDataDir() + "expected/" + "testVariantFiltration_testFilteringDPfromINFO.vcf")
+        );
+
+        spec.executeTest("test file", this);
+    }
+
+    @Test
+    public void testFilteringDPfromFORMAT() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString("filteringDepthInFormat.vcf", " -genotypeFilterExpression 'DP < 8' --genotypeFilterName lowDP "),
+                Arrays.asList(getToolTestDataDir() + "expected/" + "testVariantFiltration_testFilteringDPfromFORMAT.vcf")
+        );
+
+        spec.executeTest("test file", this);
+    }
+
+    @Test
+    public void testInvertGenotypeFilterExpression() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString("filteringDepthInFormat.vcf", " --genotypeFilterExpression 'DP < 8' --genotypeFilterName highDP --invertGenotypeFilterExpression "),
+                Arrays.asList(getToolTestDataDir() + "expected/" + "testVariantFiltration_testInvertGenotypeFilterExpression.vcf")
+        );
+
+        spec.executeTest("test file", this);
+    }
+
+    @Test
+    public void testInvertJexlGenotypeFilterExpression() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString("filteringDepthInFormat.vcf", " --genotypeFilterExpression 'DP >= 8' --genotypeFilterName highDP "),
+                Arrays.asList(getToolTestDataDir() + "expected/" + "testVariantFiltration_testInvertJexlGenotypeFilterExpression.vcf")
+        );
```

Comment:
This test does not match its name -- it does not appear to be inverting anything.